### PR TITLE
Update actions.

### PR DESCRIPTION
**Pull request by condnsdmatters**

### CODE REVIEW

**Comment by condnsdmatters** [comment]
Key focus here is to fix the GH Actions that was breaking for building the Dockerfiles.

You should hopefully see these are fixed [here](https://github.com/facebookresearch/nle/actions/runs/3501401500)

Also I have removed the cron job and done the fixes proposed #218 which i think we paused on account of the NHC

**Comment by condnsdmatters** [comment]
Also - no idea why I wasnt getting emails for that failing CI....

File: .github/workflows/build_docker.yml [hunk 1]
```diff
@@ -40,12 +40,11 @@ jobs:
...
-        repository: ${{ format('fairnle/nle-{0}', matrix.distro) }}
-        dockerfile: ${{ format('docker/Dockerfile-{0}', matrix.distro) }}
-        tags: ${{ github.event.release.tag_name }},latest
+        file: ${{ format('docker/Dockerfile-{0}', matrix.distro) }}
+        tags: |
+          ${{ format('fairnle/nle-{0}', matrix.distro) }}:latest, 
+          ${{ format('fairnle/nle-{0}', matrix.distro) }}:${{ github.event.release.tag_name }}
         add_git_labels: true
-        push: true
+        push: true
```
Comment (heiner) [inline .github/workflows/build_docker.yml:50]: Triggered

**Review by heiner — APPROVED** [review]

File: .github/workflows/build_docker.yml (hunk 1 above)
Comment (condnsdmatters) [reply .github/workflows/build_docker.yml:50]:
```suggestion
        push: true
```

File: docker/Dockerfile-xenial [hunk 2]
```diff
@@ -5,6 +5,7 @@ FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu16.04
 ARG PYTHON_VERSION=3.8
 ENV DEBIAN_FRONTEND=noninteractive
 
+
```
Comment (condnsdmatters) [inline docker/Dockerfile-xenial:8]:
```suggestion
```

File: docker/Dockerfile-focal [hunk 3]
```diff
@@ -1,9 +1,11 @@
 # -*- mode: dockerfile -*-
 
-FROM nvidia/cuda:11.0-devel-ubuntu20.04
+FROM nvidia/cuda:11.4.3-devel-ubuntu20.04
 
 ARG PYTHON_VERSION=3.8
 ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/3bf863cc.pub
+
```
Comment (condnsdmatters) [inline docker/Dockerfile-focal:8]:
```suggestion
```

**Review by samvelyan — APPROVED** [review]

**Pull request merged**